addons/im_livechat/report/im_livechat_report_operator.py

```diff
@@ -35,12 +35,11 @@ def init(self):
                     C.create_date as start_date,
                     EXTRACT('epoch' FROM (max((SELECT (max(M.create_date)) FROM mail_message M JOIN mail_message_mail_channel_rel R ON (R.mail_message_id = M.id) WHERE R.mail_channel_id = C.id))-C.create_date)) as duration,
                     EXTRACT('epoch' from ((SELECT min(M.create_date) FROM mail_message M, mail_message_mail_channel_rel R WHERE M.author_id=P.id AND R.mail_channel_id = C.id AND R.mail_message_id = M.id)-(SELECT min(M.create_date) FROM mail_message M, mail_message_mail_channel_rel R WHERE M.author_id IS NULL AND R.mail_channel_id = C.id AND R.mail_message_id = M.id))) as time_to_answer
```

Comment:
This is computed based on the message from the operator. Actually, this is not working when the visitor is logged (it currently search on the anonymous message).
Now, we can refine this by exluding the message where author_id != operator_livechat_id \0/
But we might need to LEFT JOIN 2 times this tables (one with all message, one with only the visitor messages).